Contributing.md

```diff
@@ -94,7 +94,7 @@ Once we've discussed your changes and you've got your code ready, make sure that
 ## Check code style and test case style
 
 * code style
-    * Please take a look at [OpenResty Lua Coding Style Guide](CODE_STYLE.md).
+    * Please take a look at [Lua Coding Style Guide](https://github.com/luarocks/lua-style-guide).
```

Comment:
we do not use `luarocks Lua style`, the indented is 3 spaces. 

but we are using 4 spaces.

Reply:
we can still using https://github.com/apache/incubator-apisix/blob/v1.3/CODE_STYLE.md, and remove all `openresty` in this file.

Reply:
fixed by https://github.com/apache/incubator-apisix/pull/1874

Reply:
ok